src/main/java/uk/gov/hmcts/reform/blobrouter/data/notifications/NotificationEnvelope.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.reform.blobrouter.data.notifications;
+
+public class NotificationEnvelope {
```

Comment:
miho not clear name. is this  what's going to be sent to the queue?

Reply:
if not, maybe `RejectedEnvelope`?

Reply:
renamed as `RejectedEnvelope`

Reply:
https://github.com/hmcts/blob-router-service/blob/master/src/main/java/uk/gov/hmcts/reform/blobrouter/servicebus/notifications/model/NotificationMsg.java 
this will be sent to the queue